static/js/brand-store/Settings/Settings.tsx

```diff
@@ -19,30 +19,58 @@ import NotAuthorized from "../NotAuthorized";
 import PasswordToggle from "../shared/PasswordToggle";
 import SectionNav from "../SectionNav";
 
+declare global {
+  interface Window {
+    CSRF_TOKEN: string;
+  }
+}
+
+interface RootState {
+  currentStore: {
+    loading: Boolean;
+  };
+
+  members: {
+    loading: Boolean;
+  };
+}
+
+interface RouteParams {
```

Comment:
Unless there is a reason to use `interface` we typically use `type`. There are some legitimate reasons to use `interface` but I don't see them here.

Reply:
Also this type will very likely need to be shared across a number of components, you'll probably want to move this out into the file where the routes are defined (then exported), or a shared types file.